### DISCUSSION TITLE
Button Fix

### DISCUSSION THREAD

**Discussion by QaahirStewart** [unocss/unocss#1274] [General]
I have a @import "@unocss/reset/tailwind.css"; in my App Styles Nuxt 3. When it comes to using button types, the background color disappears. For a Vue app i have, i was able to solve this by reordering my CSS imports but for this Nuxt 3 Project Unocss is the only one i have. Need a fix for this if anyone experienced this issue.

**Comment by siddiquiaffan** [comment]
I'm also facing the same issue. Did you find a fix?

**Reply by zyyv** [reply]
Please use  
```css
import '@unocss/reset/tailwind-compat.css'
```
Refer to https://unocss.dev/guide/style-reset#tailwind-compat.
